code/game/machinery/doors/airlock.dm

```diff
@@ -75,15 +75,15 @@ var/list/airlock_overlays = list()
 	var/stripe_color = null
 	var/symbol_color = null
 
-	var/fill_file = 'icons/obj/doors/station/fill_steel.dmi'
```

Comment:
what are all of these icon changes, half of them have nulled out, rather than replaced

Reply:
They don't work with the door design.